app/models/copy.rb

```diff
@@ -0,0 +1,12 @@
+class Copy < ApplicationRecord
+  belongs_to :book
+  belongs_to :borrower, class_name: 'User', optional: true, foreign_key: :'borrower_id'
```

Comment:
Also because you're not just using the table name here it's a good idea to set `inverse_of`. I think in this case it would be `inverse_of: :borrowed_copies` and then `inverse_of: :borrower` on the User class.